shongo-controller/src/main/java/cz/cesnet/shongo/controller/booking/request/AbstractReservationRequest.java

```diff
@@ -385,6 +393,24 @@ public void setReusement(ReservationRequestReusement reusement)
         this.reusement = reusement;
     }
 
+    /**
+     * @return {@link #auxData}
+     */
+    @Type(type = "jsonb")
+    @Column(name = "aux_data", columnDefinition = "text")
+    public String getAuxData()
```

Comment:
Máš to ako string. Na string nepotrebuješ jsonb. Ten využiješ keby si mal JsonNode, alebo List<Child>, a iné. 
Naviac tam máš aj columDefinition z JPA a aj hibernate @Type. :)

Reply:
Jo a miesto toho možeš použiť DB agnostic @Clob od JPA. Ten by mal fungovať dobre s TEXT type v Postgresql.

Reply:
Ako `String` to je iba kvoli tomu, ze API malo s `JsonNode` problemy (v DB je to skutocne `jsonb`). Preto tam musi byt aj `@Type`. Zatial som to teda urobil, ze entita ma `JsonNode`, API ma `String` a prevadza sa to v ramci `fromApi()`/`toApi()`.

Reply:
Ok, tak aspoň odstrániť ten columnDefinition = "text".

Reply:
Ten tam zase musi byt kvoli testom, pretoze HSQLDB nema `jsonb`.